src/typography/_headings_config.scss

```diff
@@ -2,6 +2,7 @@
   @include verlag-black;
```

Comment:
since we are using `heading` as the base class. can we change this mixin to follow the same terminology `heading` or `headings`